docker/Dockerfile.deps

```diff
@@ -21,8 +21,9 @@ RUN rm -f /opt/python/cp27-cp27m
 # Boost
 RUN BOOST_VERSION=1.66.0 && \
     cd /usr/local && \
-    curl -L https://dl.bintray.com/boostorg/release/1.66.0/source/boost_${BOOST_VERSION//./_}.tar.gz | tar -xzf - && \
-    ln -s ../boost_${BOOST_VERSION//./_}/boost include/boost
+    wget https://github.com/boostorg/boost/archive/boost-${BOOST_VERSION}.tar.gz && \
+    tar -xzf boost-${BOOST_VERSION}.tar.gz && \
```

Comment:
Remove this tar so it doesn't occupy space in Docker layer.

Reply:
That should be fixed.